### PR TITLE
Add Ars Technica

### LINKED ISSUE #8701: Icon request for Ars Technica - arstechnica.com
### Brand Name

Ars Technica

### Website

https://www.arstechnica.com/

### Popularity Metric

Similarweb rank is `#5,550` (and `#53` in its category, "Computers, Electronics and Technology")
https://www.similarweb.com/website/arstechnica.com/

#### Official resources:
- Brand Style Guide: https://brandingguide.app/brand/ars-technica
- Logo, round icon style:  
https://brandingguide.app/storage/u-ploads/wKN78x61fTTvs4509vHLPNLkwfg8bxHhLstv9Ky9.png
- All Logos:  
https://brandingguide.app/brand/ars-technica/assets
- Primary Color: #ff4e00 International Orange
- Accent Color: #01cd74 Caribbean Green

#### Unofficial:
Wikipedia hosts an SVG version of the icon here: https://en.wikipedia.org/wiki/Ars_Technica#/media/File:Ars_Technica_logo_(2016).svg

### Additional Comments

Ars Technica is a 25-year-old news site in the technology, science, policy, & tech culture space

### Contributing

Yes

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
arstechnica

**Issue:** closes #8701 

**Similarweb rank:** 5,582. See https://www.similarweb.com/website/arstechnica.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from website header https://arstechnica.com
Color: `#FF4E00` taken from branding page https://brandingguide.app/brand/ars-technica

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1110,6 +1110,12 @@
             "source": "https://www.arm.com",
             "guidelines": "https://www.arm.com/company/policies/trademarks/guidelines-corporate-logo"
         },
+        {
+            "title": "Ars Technica",
+            "hex": "FF4E00",
+            "source": "https://arstechnica.com",
+            "guidelines": "https://brandingguide.app/brand/ars-technica"
+        },
```
Comment (adamrusted) [inline _data/simple-icons.json:1118]:
```suggestion
        {
            "title": "Ars Technica",
            "hex": "FF4E00",
            "source": "https://arstechnica.com"
        },
```

I would remove the 'guidelines' link - as I can't see any official affiliation with Ars Technica.

**Review by adamrusted — CHANGES_REQUESTED** [review]
Icon itself looks to be a perfect match, thank you @PratyushJayachandran 🎉
Just a minor change to the JSON before we merge.

File: _data/simple-icons.json (hunk 1 above)
Comment (PratyushJayachandran) [reply _data/simple-icons.json:1118]: I remember as it being a redirect from their official website. I cannot find it anymore so I'll remove the guidelines part.

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @PratyushJayachandran 🎉

**Pull request merged**